src/main/java/com/owncloud/android/datamodel/ThumbnailsCacheManager.java

```diff
@@ -440,7 +440,7 @@ private String getImageKey() {
 
     public static class ThumbnailGenerationTask extends AsyncTask<ThumbnailGenerationTaskObject, Void, Bitmap> {
         private final WeakReference<ImageView> mImageViewReference;
-        private static Account mAccount;
+        private User mAccount;
```

Comment:
```suggestion
        private User user;
```

Then type and name matches :)

Reply:
@tobiasKaminsky done